build.sbt

```
@@ -20,12 +20,13 @@ val natchezVersion = "0.3.7"
 lazy val gpgPass = Option(System.getenv("GPG_KEY_PASSWORD"))
 
 ThisBuild / scalafixDependencies += "org.typelevel" %% "typelevel-scalafix" % "0.5.0"
+ThisBuild / sbtPluginPublishLegacyMavenStyle := false
 
 lazy val patchVersion = scala.io.Source.fromFile("patch_version.txt").mkString.trim
 
 credentials += Credentials(
-  "OSSRH Staging API Service",
-  "ossrh-staging-api.central.sonatype.com",
+  "Sonatype Nexus Repository Manager",
+  "s01.oss.sonatype.org",
```

```suggestion
  "central.sonatype.com",
```
I think host here needs to match repo url